README.md

```diff
@@ -190,7 +254,7 @@ To create 200 cases in various states with various responders for search testing
 ```
 rake seed:search:data
 ```
-
+It appears that redis needs to be running to attempt this task - but it doesn't currently work for unknown reasons.
```

Comment:
Think we looked at this, it works when redis is running and s3 is set up, but fails on late closed cases due to the late team id